adoption_df.py

```diff
@@ -0,0 +1,54 @@
+import pandas as pd
+import numpy as np
+from .embedding_model import EmbeddingModel
+from .translation_model import TranslationModel
+from .pos_model import PosModel
+from sklearn.metrics.pairwise import cosine_similarity
+
+import warnings
+from transformers import logging
+logging.set_verbosity_error()
+warnings.simplefilter("ignore")
+
+
+class AdoptionDF():
+    def __init__(self, name: str, initial: bool):
+        if initial:
+            self.df = pd.read_csv(name)
+            self.translation = TranslationModel()
+            self.embedding = EmbeddingModel()
+            self.pos  = PosModel()
+        else: 
+            self.df = self.read_from_csv(name)
+
+
+    def update_df(self):
+        self.df = self.translation.english_of_column(self.df, 'Discription')
+        self.df = self.pos.pos_of_column(self.df)
+        self.df = self.embedding.embed_of_column(self.df)
```

Comment:
it is more common to pass the variable(df) to the class and then the function inherits it with self, you pass the ardument to the function when nothing is passed to the class